airbyte-ci/connectors/pipelines/README.md

```diff
@@ -534,12 +544,12 @@ For poetry packages, the package name and version can be taken from the `pyproje
 
 #### Options
 
-| Option                    | Required | Default                 | Mapped environment variable | Description                                                                                              |
-| ------------------------- | -------- | ----------------------- | --------------------------- | -------------------------------------------------------------------------------------------------------- |
-| `--publish-name`          | False    |                         |                             | The name of the package. Not required for poetry packages that define it in the `pyproject.toml` file    |
-| `--publish-version`       | False    |                         |                             | The version of the package. Not required for poetry packages that define it in the `pyproject.toml` file |
-| `--python-registry-token` | True     |                         | PYTHON_REGISTRY_TOKEN       | The API token to authenticate with the registry. For pypi, the `pypi-` prefix needs to be specified      |
-| `--registry-url`          | False    | https://pypi.org/simple |                             | The python registry to publish to. Defaults to main pypi                                                 |
+| Option                    | Required | Default                         | Mapped environment variable | Description                                                                                              |
+| ------------------------- | -------- | ------------------------------- | --------------------------- | -------------------------------------------------------------------------------------------------------- |
+| `--publish-name`          | False    |                                 |                             | The name of the package. Not required for poetry packages that define it in the `pyproject.toml` file    |
+| `--publish-version`       | False    |                                 |                             | The version of the package. Not required for poetry packages that define it in the `pyproject.toml` file |
+| `--python-registry-token` | True     |                                 | PYTHON_REGISTRY_TOKEN       | The API token to authenticate with the registry. For pypi, the `pypi-` prefix needs to be specified      |
+| `--registry-url`          | False    | https://upload.pypi.org/legacy/ | PYTHON_REGISTRY_URL         | The python registry to publish to. Defaults to main pypi                                                 |
```

Comment:
I'm wondering where the default URL should live in. I'd say it should be at the airbyte-ci CLI level, which would mean the `PYTHON_REGISTRY_URL` env var  does not need to be set an `main-release`

Reply:
Definitely on the airbyte-ci level, but IMHO it makes sense here as well for local usage.